features/scaffold-package-readme.feature

```diff
@@ -56,6 +56,32 @@ Feature: Scaffold a README.md file for an existing package
       Success: Uninstalled package.
       """
 
+  Scenario: Scaffold a README.md based with custom repository branch
+    Given an empty directory
+
+    When I run `wp package path`
+    Then save STDOUT as {PACKAGE_PATH}
+
+    When I run `wp scaffold package wp-cli/custom-branch`
+    And I run `wp scaffold package-readme --branch=custom --force`
```

Comment:
```suggestion
    And I run `wp scaffold package-readme custom-branch --branch=custom --force`
```

Reply:
This chenge don't fix this test https://github.com/wp-cli/scaffold-package-command/runs/7766330371?check_suite_focus=true

Reply:
But you did not exactly apply my suggestion in 4420edbf893ef580c86dce1e5da0ae23df85d8f1


`wp scaffold package custom-branch` does not work. That's why there are these `Error: 'custom-branch' is an invalid package name. Package scaffold expects '<author>/<package>'.`errors right now
 

You should keep it like that: `wp scaffold package wp-cli/custom-branch`

Reply:
fixed this is https://github.com/wp-cli/scaffold-package-command/pull/206/commits/0cf7e2e29eca6cc8d30896710b5f3950c220fa0e but test is failing still

Reply:
I haven't tested it really, but since you use `wp scaffold package` for this new test, where does `wp scaffold package wp-cli/custom-branch` create a file? You'd need to update the `wp scaffold package-readme` call accordingly.

None of the other tests use `wp scaffold package`  though, so maybe just rewrite the test to not use it?

Reply:
My test is adapted version of https://github.com/wp-cli/scaffold-package-command/blob/e5a4b7b51c5b43dc9c1d8c707d58224fde80311f/features/scaffold-package-readme.feature#L29-L57

Reply:
Then maybe `wp scaffold package-readme {PACKAGE_PATH}/local/wp-cli/custom-branch--branch=custom --force`?

Reply:
There we go :-)